deploy/kubeturbo-operator/certified-operator-automation-1.py

```diff
@@ -0,0 +1,178 @@
+import os
+import shutil
+from github import Github
+import git
+import re
+import docker
+
```

Comment:
In general, I recommend to create different functions for different steps. This way those functions will be used or extended for other certifications.